README.md

```diff
@@ -109,14 +112,15 @@ jobs:
 | token             | ✅       | The GitHub access token                                                                                                                              |                       |
 | ticketPrefix      |          | The unique identifier for the ticket/issue                                                                                                           | #                     |
 | titleFormat       |          | The intended format the title should be set to if it doesn't match the regular expression. Available variables are `%prefix%`, `%id%`, and `%title%` | %prefix%%id%: %title% |
-| titleRegex        |          | The regular expression used to search the title for the intended format                                                                              | ^#(\d+)               |
+| titleRegex        |          | The regular expression used to search the title for the intended format                                                                              | ^(CH)(-?)(\d{3,})     |
 | titleRegexFlags   |          | The regular expression flags applied to the title regular expression                                                                                 | gi                    |
-| bodyRegex         |          | The regular expression used to search the body for a shorthand reference (example `#123`)                                                            | #(\d+)                |
+| branchRegex       |          | The regular expression used to search the branch for the intended format                                                                             | ^(CH)(-?)(\d{3,})     |
+| branchRegexFlags  |          | The regular expression flags applied to the branch regular expression                                                                                | gi                    |
+| bodyRegex         |          | The regular expression used to search the body for a shorthand reference (example `#123`)                                                            | (CH)(-?)(\d{3,})      |
 | bodyRegexFlags    |          | The flags applied to the body regular expression when searching for a shorthand reference                                                            | gim                   |
 | bodyURLRegex      |          | The regular expression used to search the body for a URL reference (example `https://github.com/octocat/hello-world/issues/1`)                       |                       |
 | bodyURLRegexFlags |          | The flags applied to the body regular expression when searching for a URL reference                                                                  | gim                   |
 
-
 ## Caveat
```

Comment:
I don't actually think this is true (anymore?). Just updating the title on GitHub will rerun the check. So you can probably just remove this section.